data/garden/reference/countries_regions.csv

```diff
@@ -2,7 +2,7 @@ code,name,iso_alpha2,iso_alpha3,imf_code,cow_letter,cow_code,unctad_code,marc_co
 AFG,Afghanistan,AF,AFG,512.0,AFG,700.0,AFG,AF,AFGN,AFG,AFG,http://www.wikidata.org/entity/Q889,Afghanistan,15.0,562.0,,
```

Comment:
I don't know how this happened, but I don't think we should add country codes as aliases. The aliases are for country names that are called different things (e.g. "United States", "United States of America").